### PR TITLE
Sort search results by date from newest to oldest and oldest to newest

### PR DESCRIPTION
**Pull request by tilthouse**
Connected to https://github.com/yalelibrary/YUL-DC/issues/242

Ticket: https://app.zenhub.com/workspaces/yul-dc-5e50083561915b11fc9e5850/issues/yalelibrary/yul-dc/242

### CODE REVIEW

**Comment by tilthouse** [comment]
Can change tests to test ordering of page.body.index If necessary

File: spec/system/sort_in_search_spec.rb [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    click_on 'Search'
+    click_on 'Sort by relevance'
+    click_on 'date (oldest first)'
+
+    content = find(:css, '#content')
+    expect(content).to have_content('1. Aquila Eccellenza')
+    expect(content).to have_content('2. HandsomeDan Bulldog')
+    expect(content).to have_content('3. Amor Llama')
+    expect(content).to have_content('4. Rhett Lecheire')
+  end
```
Comment (alishaevn) [inline spec/system/sort_in_search_spec.rb:81]: 2 of the rubocop errors show `Example has too many lines`. In this case, it feels like all of these lines are needed though. Would this be a good case for changing the rubocop rule?

Comment (bess) [reply spec/system/sort_in_search_spec.rb:81]: @alishaevn Yes, I do think that's a good case for either changing the rubocop rule or adding a specific override.

**Comment by tilthouse** [comment]
Screen Shot 2020-06-22 at 10 02 43 AM
Screen Shot 2020-06-22 at 10 03 11 AM

**Pull request merged**